Header header
trajectory_msgs/JointTrajectory trajectory
string[] contact_joint_names
JointContactTrajectoryContacts[] contacts
